### PR TITLE
polish deepCF model to support real dataset

### PR DESCRIPTION
**Pull request by panyx0718**

### CODE REVIEW

File: python/paddle/fluid/initializer.py [hunk 1]
```diff
@@ -756,7 +756,7 @@ def __call__(self, var, block):
             values = [int(v) for v in self._value.flat]
         else:
             raise ValueError("Unsupported dtype %s", self._value.dtype)
-        if self._value.size > 1024 * 1024 * 5:
+        if self._value.size > 1024 * 1024 * 1024:
```
Comment (JiabinYang) [inline python/paddle/fluid/initializer.py:759]: why change this?

**Review by JiabinYang — COMMENTED** [review]
one question

File: python/paddle/fluid/initializer.py (hunk 1 above)
Comment (panyx0718) [reply python/paddle/fluid/initializer.py:759]: NumpyArrayInitializer limits numpy size. I think a few GBs is OK

**Review by JiabinYang — APPROVED** [review]
LGTM

**Pull request merged**